src/loaders/swc-loader.js

```diff
@@ -0,0 +1,110 @@
+// Source code from https://github.com/swc-project/swc-loader/blob/master/src/index.js
+// with the following changes:
+// - swapped out `@swc/core` for `@swc/wasm`
```

Comment:
Any reason to use `@swc/wasm` instead of `@swc/core`? Seems like this was the cause of the slowdown that @cb1kenobi mentioned in https://github.com/vercel/ncc/pull/926#issuecomment-1198891776

Reply:
Yes, see comment (https://github.com/vercel/ncc/pull/926#issuecomment-1198902746) for the explanation.

Honestly, I do not know the exact reason. Previous work in here was using it (https://github.com/vercel/ncc/pull/777/files#diff-8c08ac17cfa691b1772e338822ad7b9cada2a78c0851239d4af55c561c943001R3), and per https://github.com/vercel/ncc/pull/777/files#diff-a992dd64a8c626fedc95d613f43602b91f6dc6339893afaf405dc05fa13c9ceaR96 I only could guess per-platform native binaries are not working well with how ncc works.

Reply:
FWIW, I rebooted and pulled the latest and build times have dropped from 10 seconds down to 5.5 seconds on average. Mind you the latest ncc release builds in around 2 seconds, so there is definitely a performance regression.